src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -5,905 +5,908 @@
 /// .NET Core + CLI tools helpers
 module Fake.DotNet.Cli
```

Comment:
Can we remove the `Cli` here? The idea is that people only need to open `Fake.Dotnet` and have all the relevant modules in scope (otherwise we have a lot of namespace/module `open` at each fake script.

But I think this is what doesn't work :/

Reply:
If it doesn't work we might consider using `[<AutoOpen>]` here. Normally we don't use `AutoOpen` anymore but in this scenario it might be a viable workaround (as long as we make sure only sub-modules are contained and no variables/functions).

Reply:
I will try to update the module path and see the result.

I am not sure to understand where you want to place `[<AutoOpen>]` if needed.

Reply:
Basically first try to remove ‘.Cli’ from the commented line. Only if that doesn’t work we can put autoopen above the commented line. The effect should be the same for both solutions...

Reply:
But don’t worry, I can do this after merging if needed